TestApp/Scripts/experience.js

```diff
@@ -10,14 +10,16 @@ function CreateBoxAsync() {
 }
 
 function CreateSpheresAsync() {
```

Comment:
Revert changes to this function.

Reply:
Also most of the rest of the changes to this file. Make the OpenXR thing an option similar to existing options.